### PR TITLE
Speed Test updates

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2022-01-28 at 5 51 11 PM
Screen Shot 2022-01-28 at 5 50 26 PM
Screen Shot 2022-01-28 at 5 50 24 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Speedtest/Speedtest.tsx [hunk 1]
```diff
@@ -212,21 +200,74 @@ const Speedtest = ({ classes, distributedSetup }: ISpeedtest) => {
...
+                      noLabelMinWidth={true}
+                      value={size}
+                      disabled={start}
+                      overlayObject={
+                        <InputUnitMenu
+                          id={"size-unit"}
+                          onUnitChange={setSizeUnit}
+                          unitSelected={sizeUnit}
+                          unitsList={[
+                            { label: "KB", value: "KB" },
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Speedtest/Speedtest.tsx:247]: any specific reason for using `KB` , `MB` `GB` Units, will they be `KiB`, `MiB`, `GiB` as in other places.?

File: portal-ui/src/screens/Console/Speedtest/Speedtest.tsx [hunk 2]
```diff
@@ -212,21 +200,74 @@ const Speedtest = ({ classes, distributedSetup }: ISpeedtest) => {
...
+                    <ProgressBarWrapper
+                      value={speedometerValue}
+                      ready={currStatus !== null && !start}
+                      indeterminate={start}
+                      size={"small"}
+                    />
+                  </div>
+                </Grid>
+                <Grid item xs={4}>
+                  <div style={{ width: 300 }}>
```
[the remark this answers is not in the document]
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Speedtest/Speedtest.tsx:230]: done

File: portal-ui/src/screens/Console/Speedtest/SpeedTestUnit.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+//  This file is part of MinIO Console Server
+//  Copyright (c) 2021 MinIO, Inc.
```
[the remark this answers is not in the document]
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Speedtest/SpeedTestUnit.tsx:2]: changed

File: portal-ui/src/screens/Console/Speedtest/Speedtest.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Speedtest/Speedtest.tsx]: no, we should standarize on `MiB` and such, I updated it

**Review by Alevsk — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**